msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx

```diff
@@ -1576,4 +1576,17 @@
         <comment>RuntimeIdentifier: don't translate (it's the name of a MSBuild property)</comment>
     </data>
 
+    <data name="E7125" xml:space="preserve">
+        <value>Unable to parse the value '{0}' for the property '{1}.</value>
+        <comment>
+            {0}: user-provided value
+            {1]: name of an MSBuild property
```

Comment:
You probably want this to be `{1}` and not `{1]`

Reply:
@stephen-hawley Fixed!